packages/file_selector/file_selector_windows/example/lib/open_multiple_images_page.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that shows an example of openFiles
```

Comment:
Maybe something like "Screen that allows the user to select multiple files, then displays the selected images in a gallery dialog."